index.html

```diff
@@ -32,36 +34,38 @@ <h1 class="logo">Relaxr</h1>
 				<a class="button yellow" href="#">Get it Now</a>
 			</div>
 		</section>
-		<section id="benefits">
-			<div class="container">
-				<h3>Benefits</h3>
-				<p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+		<div class="col-med-12">
+			<section id="benefits">
+				<div class="container">
```

Comment:
We typically wouldn't put a bootstrap "container" inside of a column. Generally, the container is the outermost element of the grid.